mappings/net/minecraft/entity/MobWithInventory.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_6067 net/minecraft/entity/MobWithInventory
```

Comment:
Maybe like `InventoryOwner` or `InventoryHolder`? I don't see how this is related to `Mob`.

Reply:
I mean it's only used by MerchantEntity, WanderingTraderEntity and PiglinEntity

Reply:
Really need to see where that getInventory method is used. Also that method has a weird annotation